[2022-04-22 13:41:23,860] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: cso_2295_ipo_2_cso_xls.py.log_sensor3 manual__2022-04-22T13:40:26.332001+00:00 [queued]>
[2022-04-22 13:41:23,878] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: cso_2295_ipo_2_cso_xls.py.log_sensor3 manual__2022-04-22T13:40:26.332001+00:00 [queued]>
[2022-04-22 13:41:23,878] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-04-22 13:41:23,878] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-04-22 13:41:23,879] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-04-22 13:41:23,969] {taskinstance.py:1264} INFO - Executing <Task(SqlSensor): log_sensor3> on 2022-04-22 13:40:26.332001+00:00
[2022-04-22 13:41:23,977] {standard_task_runner.py:52} INFO - Started process 13755 to run task
[2022-04-22 13:41:23,980] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cso_2295_ipo_2_cso_xls.py', 'log_sensor3', 'manual__2022-04-22T13:40:26.332001+00:00', '--job-id', '1027', '--raw', '--subdir', 'DAGS_FOLDER/cso_2295/cso_2295_ipo_2_cso_xls.py', '--cfg-path', '/tmp/tmpus5ubhck', '--error-file', '/tmp/tmpoxgsx32p']
[2022-04-22 13:41:23,981] {standard_task_runner.py:77} INFO - Job 1027: Subtask log_sensor3
[2022-04-22 13:41:24,104] {logging_mixin.py:109} INFO - Running <TaskInstance: cso_2295_ipo_2_cso_xls.py.log_sensor3 manual__2022-04-22T13:40:26.332001+00:00 [running]> on host 22e04adb6bfa
[2022-04-22 13:41:24,359] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cso_2295_ipo_2_cso_xls.py
AIRFLOW_CTX_TASK_ID=log_sensor3
AIRFLOW_CTX_EXECUTION_DATE=2022-04-22T13:40:26.332001+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-22T13:40:26.332001+00:00
[2022-04-22 13:41:24,401] {base.py:79} INFO - Using connection to: id: postgres_default. Host: postgres, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2022-04-22 13:41:25,481] {sql.py:98} INFO - Poking: select case when q = 'true' then 1 else 0 end as q from  (
	select case when count(id)=0 then 'true' else 'false' end as q
	from list_flow_log
	where object_flow = 0 and status = 1
) t1; (with parameters None)
[2022-04-22 13:41:25,535] {base.py:79} INFO - Using connection to: id: postgres_default. Host: postgres, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2022-04-22 13:42:25,633] {base.py:79} INFO - Using connection to: id: postgres_default. Host: postgres, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2022-04-22 13:42:25,693] {sql.py:98} INFO - Poking: select case when q = 'true' then 1 else 0 end as q from  (
	select case when count(id)=0 then 'true' else 'false' end as q
	from list_flow_log
	where object_flow = 0 and status = 1
) t1; (with parameters None)
[2022-04-22 13:42:25,699] {base.py:79} INFO - Using connection to: id: postgres_default. Host: postgres, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2022-04-22 13:42:35,144] {local_task_job.py:212} WARNING - State of this instance has been externally set to skipped. Terminating instance.
[2022-04-22 13:42:35,145] {process_utils.py:124} INFO - Sending Signals.SIGTERM to group 13755. PIDs of all processes in the group: [13755]
[2022-04-22 13:42:35,146] {process_utils.py:75} INFO - Sending the signal Signals.SIGTERM to group 13755
[2022-04-22 13:42:35,146] {taskinstance.py:1413} ERROR - Received SIGTERM. Terminating subprocesses.
[2022-04-22 13:42:35,172] {process_utils.py:70} INFO - Process psutil.Process(pid=13755, status='terminated', exitcode=0, started='13:41:23') (13755) terminated with exit code 0
